front_end/content.py

```diff
@@ -203,6 +203,19 @@ def save_problem(problem_basics, problem_details):
     write_file(convert_dict_to_yaml(basics_to_save), get_problem_dir_path(problem_basics["assignment"]["course"]["id"], problem_basics["assignment"]["id"], problem_basics["id"]) + "basics")
     write_file(convert_dict_to_yaml(problem_details), get_problem_dir_path(problem_basics["assignment"]["course"]["id"], problem_basics["assignment"]["id"], problem_basics["id"]) + "details")
 
+def get_next_submission_title(course, assignment, problem, user):
```

Comment:
This is nitpicky, but please change the name of this function to get_next_submission_id. I've been using the term "title" to refer to something that instructors enter, whereas this is a computer-generated identifier.

Reply:
Done!